sim/SIM-PROTOCOL.md

```diff
@@ -11,6 +11,8 @@ Receiving messages
 
 The beginning of a battle will look something like this:
 
+    |elo|Anonycat|1200
```

Comment:
I feel like this should come after `|player|` introduces the player, or perhaps it should be combined with `|player|` (but turned into a separate line in the output logs)